delta-api/src/main/java/io/cdap/delta/api/Sequenced.java

```diff
@@ -26,20 +26,28 @@
 public class Sequenced<T> {
   private final T event;
   private final long sequenceNumber;
+  private static final long UNSUPPORTED_SEQUENCE = -1;
 
+  public Sequenced(T event) {
+    this(event, UNSUPPORTED_SEQUENCE);
+  }
   public Sequenced(T event, long sequenceNumber) {
     this.event = event;
     this.sequenceNumber = sequenceNumber;
   }
 
-  public T getEvent() {
-    return event;
-  }
-
   public long getSequenceNumber() {
```

Comment:
add javadoc about when it will throw an exceptipn?

Reply:
done :)